<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectInformation</class>
 <widget class="QDialog" name="ProjectInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>205</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>308</width>
    <height>205</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>308</width>
    <height>205</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Project Information</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F7F7F7;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: Bold 13px &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Survey Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSurveyType">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13px &quot;Segoe UI&quot;;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>WBPHS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblFileName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13px &quot;Segoe UI&quot;;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>sample.asc</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: Bold 13px &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Export Filename</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDirectory">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13px &quot;Segoe UI&quot;;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>C:/Users/file/path/documents/fws/scribe/fold er/reports/</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: Bold 13px &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Project Directory</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>308</width>
     <height>56</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>308</width>
     <height>56</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>308</width>
     <height>56</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #EEEEEE;
opacity: 1;
margin: 0;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>53</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>53</width>
            <height>26</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
  border: 1px solid #ADADAD;
  opacity: 1;
  text-align: center;
  font: BOLD 13px &quot;Segoe UI&quot;;
  color: #000000;
  background: #E1E1E1;
  margin: 0;
  padding-bottom:4px;
}
QPushButton:hover{
  color: #000000;
  background-color: #E5F1FB;
  border-color: #0078D7;
}
QPushButton:focus{
  #box-shadow: none, 0 0 0 0.2rem rgba(0, 188, 212, 0.5);
}
QPushButto:disabled {
  color: #ffffff;
  background-color: #00bcd4;
  border-color: #00bcd4;
}</string>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>12</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
